packages/react-native-editor/bin/generate-pot-files.sh

```diff
@@ -191,7 +191,7 @@ fi
 # Define constants
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 GUTENBERG_SOURCE_CODE_DIR="$SCRIPT_DIR/../../.."
-WP_CLI="php -d memory_limit=4G $SCRIPT_DIR/wp-cli.phar"
+WP_CLI="php -d memory_limit=4G -d error_reporting=E_ALL&~E_DEPRECATED $SCRIPT_DIR/wp-cli.phar"
```

Comment:
I noticed when using PHP 8.2.2 that WP-CLI displays logs about deprecated functions. Since this message doesn't provide helpful information for the purpose of the translation pipeline, I decided to filter them out.